doc/source/serve/api/index.md

```diff
@@ -372,3 +372,12 @@ Content-Type: application/json
    metrics.Histogram
    metrics.Gauge
 ```
+
+# Logging Config API
```

Comment:
These sections are pretty fine-grained -- how about combine with "Metrics API" and call the section "Observability"?